Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -306,9 +267,8 @@ - (CGFloat)quotedMessageTopMargin
 
 - (void)clearQuotedMessagePreview
 {
-    // TODO animate
     if (self.quotedMessagePreview) {
-        [self.contentStackView removeArrangedSubview:self.quotedMessagePreview];
+        [self.contentRows removeArrangedSubview:self.quotedMessagePreview];
```

Comment:
I often prefer "rows" or "columns" when naming a stackview to carry just a little more information.